lib/patch_tag_helper.rb

```diff
@@ -11,30 +11,9 @@ def tag_option(key, value, escape)
     if value.is_a?(Array)
       value = escape ? safe_join(value, " ") : value.join(" ")
     else
-      value = escape ? ERB::Util.unwrapped_html_escape(value) : value.to_s
+      value = escape ? ERB::Util.html_escape(value) : value.to_s
```

Comment:
> 4.1's html_escape is identical to unwrapped_html_escape

To me, it doesn't look identical.

- `html_escape` in 4.1 [marks the gsubbed string as `html_safe`](https://github.com/rails/rails/blob/4-1-stable/activesupport/lib/active_support/core_ext/string/output_safety.rb#L25).
- `unwrapped_html_escape` in 4.2 [does not](https://github.com/rails/rails/blob/4-2-stable/activesupport/lib/active_support/core_ext/string/output_safety.rb#L41).

I'm not sure if I've misinterpreted what the commit is doing, but I'm just going to drop this commit to retain the manual patch until we upgrade to Rails 4.2 and get the patch included in the release.